superset/views/csv_import.py

```diff
@@ -101,7 +106,7 @@ def add(self) -> Response:
 
         try:
             if database_id != -1:
-                schema = None if not form_data["schema"] else form_data["schema"]
+                schema = form_data["schema"] or None
```

Comment:
Why did you remove the if?

Reply:
was proposed in the PR into superset-master, therefore I changed it here too (https://github.com/apache/incubator-superset/pull/8556/files/2ca3740d13850b18116ace1ce8c0990885ff5280#r346021226)